tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -80,6 +89,11 @@ spec:
               tag=$(jq -r '.tag' <<< $component)
           fi
 
+          if [ -n "$(params.tagPrefix)" ]; then
```

Comment:
Why do this for every component over and over again? This should be outside the `for` cycle.

Reply:
I changed the code a bit to make the logic a bit clear. 
It always needs to check if the tag to use is the one in the snapshot file (for each component) and not use it only if the tagPrefix parameter is set. The best I could do was just move the param check outside, the it still needs to check for something in the loop.

Reply:
I see, I didn't realize there was more logic for the tags. I added more comments there.